addons/sourcemod/scripting/vsh/bosses/boss_hale.sp

```diff
@@ -159,6 +178,45 @@ public void SaxtonHale_OnSpawn(SaxtonHaleBase boss)
 	*/
 }
 
+public void SaxtonHale_OnRage(SaxtonHaleBase boss)
+{
+	boss.flSpeed *= 1.3;
+	g_bHaleSpeedRage[boss.iClient] = true;
+}
```

Comment:
This might be able to be stacked and break if rage is done before current rage isn't finished